dotnet-client-libraries/WireMock.NET/WireMockNet/PlanetsService.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Net;
+using System.Text.Json;
+
+namespace WireMockNet
+{
+    public class PlanetsService
+    {
+        private readonly HttpClient _httpClient;
+
+        public PlanetsService(HttpClient httpClient)
+        {
+            _httpClient = httpClient ??
+                throw new ArgumentNullException(nameof(httpClient));
+        }
+
+        public async Task<Planet?> GetPlanetByIdAsync(int id)
+        {
+            var response = await _httpClient.GetAsync(_httpClient.BaseAddress + $"planets/{id}");
```

Comment:
No need to create the aboslute URI since the BaseAddress is already set on the HttpClient, you can just give the relative URI for the endpoint.